Assets/StreamingAssets/Localization/ru_RU.lang

```diff
@@ -1,18 +1,30 @@
+﻿menu_construction=Строительство
+menu_work=Работы
+menu_world=Мир
+menu_options=Главное Меню
+menu_settings=Настройки
+menu_furniture=Мебель
```

Comment:
At this stage everything is a furniture. It's just a placeholder, mostly to tick the box on wiki so that it's not auto-translated.